common/source/docs/common-simple-object-avoidance.rst

```diff
@@ -0,0 +1,108 @@
+
+In ALTHOLD mode, the aggregate threat is translated into an attempt to add lean input into the pilot's commands to move the Copter away from the aggregate obstructions. The pilot can still overcome these additions to his command inputs and fly into a object, if determined to do so.
+
+In LOITER, either stopping in front of the object or a "sliding" algorithm is used to avoid it. "Sliding" involves slowing and slightly yawing as the vehicle approaches the object. For example, if Copter approaches a fence line at an angle, it will "slide along" the fence as the operator pushes the Copter toward the fence. Head-on approaches would stop, even if the pilot continues to "pushes" it forward.
+
+Setup the Proximity Sensor
+==========================
+
+For 360 lidar follow the instructions corresponding to the lidar on the vehicle
+
+  - :ref:`Lightware SF40C <common-lightware-sf40c-objectavoidance>`
+  - :ref:`TeraRanger Tower/ Tower EVO <common-teraranger-tower-objectavoidance>`
+  - :ref:`RPLidarA2/A3 <common-rplidar-a2>`
+
+For other rangefinders follow the instructions found in there individual setup pages :ref:`here <common-rangefinder-landingpage>`
+
+Be sure to read the :ref:`common-rangefinder-setup` page
+
+Configuring Simple Avoidance for Copter
+=======================================
+
+- set :ref:`AVOID_ENABLE <AVOID_ENABLE>` = 7 ("All") to use all sources of barrier information including "Proximity" sensors
+
+- in :ref:`Loiter <loiter-mode>`
+- 
+  -  set :ref:`PRX_TYPE <PRX_TYPE>` to a 360 deg Lidar type being used or = 4, to enable using range finders as "proximity sensors"
+  - :ref:`AVOID_MARGIN <AVOID_MARGIN>` controls how many meters from the barrier the vehicle will attempt to stop or try to slide along it
+  - :ref:`AVOID_BEHAVE <AVOID_BEHAVE>` allows setting whether the vehicle should simply Stop in front of the barrier or Slide around it. This parameter only affects Copter, since Rover always stops.
+
+- in :ref:`AltHold <altholdmode>`
+
+  - :ref:`AVOID_DIST_MAX <AVOID_DIST_MAX>` controls how far from a barrier the vehicle starts leaning away from the barrier
+  - :ref:`AVOID_ANGLE_MAX <AVOID_ANGLE_MAX>` controls how far the vehicle will try to lean away from the barrier
+
+Configuring Simple Avoidance for Rover
+======================================
+
+- set :ref:`AVOID_ENABLE <AVOID_ENABLE>` = 7 ("All") to use all sources of barrier information including "Proximity" sensors.
+- set :ref:`PRX_TYPE <PRX_TYPE>` = "4" to enable using range finders as "proximity sensors"
+- Rover only attempts to stop the vehicle before it hits objects in :ref:`Acro <acro-mode>`, :ref:`Steering <steering-mode>`, :ref:`Loiter <loiter-mode>` and :ref:`Guided <guided-mode>` modes
```

Comment:
Actually I think Rover will attempt to stop before hitting things in all modes.

Reply:
including manual?

Reply:
fixed to exclude manual mode